packages/web/tests/pages/InvalidEmail.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render, screen } from '../testUtils/testTools';
+import { getMock } from '../testUtils/getMock';
+import { MarketingLayout } from '../../src/components/Layout';
+import InvalidEmail from '../../src/pages/invalidEmail';
+
+jest.mock('../../src/components/Layout/MarketingLayout.tsx');
+getMock(MarketingLayout).mockImplementation(({ children }) => <>{children}</>);
+
+describe('web /invalidEmail', () => {
+  it('renders', async () => {
```

Comment:
`it renders` is insufficient; can you add more detail here to explain what this test is evaluating?